src/Concerns/Isolatable.php

```diff
@@ -25,7 +25,7 @@ protected function isolatedStatusCode(): int
 
     protected function getIsolateOption(): bool|int
     {
-        return $this->hasIsolateOption() ? $this->option(Options::ISOLATED) : false;
+        return $this->hasIsolateOption() && $this->option(Options::ISOLATED);
```

Comment:
```suggestion
        if ($code = $this->option(Options::ISOLATED)) {
            return intval($code);
        }

        return false;
```
In this sentence, the method will always return a boolean value, despite the fact that the parameter can take not only it, but also a numeric value as well.

For example, `--isolate=2`

![image](https://github.com/TheDragonCode/laravel-actions/assets/10347617/da0179d8-aae2-452c-8e5d-07e1e4216d5e)

https://laravel.com/docs/10.x/artisan#isolatable-commands